### DISCUSSION TITLE
OpenStatus vs BetterStack free plan

### DISCUSSION THREAD

**Discussion by shawnmclean** [openstatusHQ/openstatus#956] [General]
Hey guys,

I just learnt about OpenStatus and was looking to move from BetterStack free plan to your free plan.

Just because I want to support open source products.

However, the feature comparison in the plans preventing this:

1. Custom domains
2. 5 mins heart beat (doesn't matter to me)
3. Content Body check (OpenStatus doesn't give me good feedback here except the toast at the bottom saying an error occured when testing)

Just thought I'd share.

**Comment by thibaultleouay** [comment]
Hello @shawnmclean 

1 &2 - It's In our paid plan. We are a fully bootstrapped product and cannot afford to offer as much for free since we don't have VC funding. 
Compared to many other bootstrapped products, we still offer a free plan :) 

3- It's really something we need to improve thanks for the feedback

**Comment by thibaultleouay** [comment]
@shawnmclean  We have just pushed some feedback for body check 
![CleanShot 2024-07-25 at 22 44 59](https://github.com/user-attachments/assets/543d4bf1-eb6d-460b-aefc-ab7ee7c1df84)

**Comment by shawnmclean** [comment]
Nice! Confirmed fixed.
